packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -228,7 +229,7 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
     setMenuHidden(true);
   };
 
-  const participantStateString = participantStateStringTrampoline(props, strings);
+  const participantStateString = participantStateStringTrampoline(strings, participantState);
```

Comment:
issue isn't just with the string, when an ACS Teams interop participant joins they go through the following states:
`idle -> connecting -> lobby -> connected`

This means we will try to render a tile during the idle and connecting states, then remove the tile when they hit the lobby, then return the tile once the hit connected